Ratnesh/week 2 assignment 1 by ratnesh/count duplicate array.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+ 
+int main()
+{
+  int arr[10], i, j, Size, Count = 0;
```

Comment:
Define variable name with lower case

Reply:
Why? Is there is any rule? I think it is not necessary.
See:- https://mathbits.com/MathBits/CompSci/DataBasics/naming.htm